src/docs/cookbook/networking/send-data.md

```diff
@@ -166,17 +169,14 @@ user input from a `TextField`.
 When the `ElevatedButton` is pressed, the `_futureAlbum`
 is set to the value returned by `createAlbum()` method.
 
-<!-- skip -->
+<?code-excerpt "lib/main.dart (Column)" replace="/^return //g;/;$//g"?>
 ```dart
 Column(
   mainAxisAlignment: MainAxisAlignment.center,
   children: <Widget>[
-    Padding(
-      padding: const EdgeInsets.all(8.0),
```

Comment:
for some reason the final example doesn't have a padding, so I left it out

Reply:
I defer to @domesticmouse on this.

Reply:
The code looks easier to read without the `Padding` widget. 

LGTM